.env.unified.lan

```
@@ -0,0 +1,64 @@
+# LAN/Production Configuration
+# Copy to .env for LAN deployment
+
+# ========================================
+# REQUIRED: Database Connection
+# ========================================
+SUPABASE_URL=https://ywqhytytvsugasmbpdfa.supabase.co
+SUPABASE_SERVICE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Inl3cWh5dHl0dnN1Z2FzbWJwZGZhIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc1NjQyMjIxNiwiZXhwIjoyMDcxOTk4MjE2fQ.3DnxBT2hVtQgDdrxq5L2YgElZBV2-vkhUFKnJ7txQGQ
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)